### PR TITLE
AC Test Fix - LRAC-10258 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-10258 (ticket)
https://testray.liferay.com/home/-/testray/case_results?tab=runs&testrayBuildId=1598211961 (fix)

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 1]
```diff
@@ -184,10 +198,86 @@ definition {
...
+
+			ACUtils.viewNameList(nameList = "ac1 ac1");
+
+			Button.clickDone();
+
+			ACSegments.saveSegment();
+		}
+
+		task ("Asset user present in the Segments") {
+			ACNavigation.switchTab(tabName = "Membership");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:242]: hi @AfonsoHSantos , could you please fix this typo?
`task ("Asset user present in the Segments") {`
to 
`task ("Assert user present in the Segments") {`

Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 2]
```diff
@@ -184,10 +198,86 @@ definition {
...
 			ACUtils.viewNameList(nameList = "ac ac");
+
+			Button.clickDone();
+
+			ACSegments.saveSegment();
+		}
+
+		task ("Asset user present in the Segments") {
+			ACNavigation.switchTab(tabName = "Membership");
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:213]: hi @AfonsoHSantos , could you please fix this typo?
`task ("Asset user present in the Segments") {`
to
`task ("Assert user present in the Segments") {`

Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 3]
```diff
@@ -184,10 +198,86 @@ definition {
...
+			ACUtils.viewNameList(nameList = "ac ac,ac1 ac1");
+
+			Button.clickDone();
+
+			ACSegments.saveSegment();
+		}
+
+		task ("Asset users present in the Segments") {
+			ACNavigation.switchTab(tabName = "Membership");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:278]: hi @AfonsoHSantos , could you please fix this typo?
`task ("Asset users present in the Segments") {`
to
`task ("Assert that the users are present in the Segments") {`

Thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**